src/components/ContestButton/ContestButton.css

```diff
@@ -9,8 +9,7 @@
 .ContestButton.ui.button.mini:active,
 .ContestButton.ui.button.mini:focus,
 .ContestButton.ui.button.mini:hover {
-  background-image: url('../../images/contest-button.png');
-  background-size: 100%;
+  background-color: #ff2d55;
```

Comment:
This color should be taken from the CSS variable

Reply:
Will do